firefox-ios/Client/Frontend/Browser/MainMenu/MainMenuConfigurationUtility.swift

```diff
@@ -71,11 +71,12 @@ struct MainMenuConfigurationUtility: Equatable {
     ) -> [MenuSection] {
         // Always include these sections
         var menuSections: [MenuSection] = [
-            getNewTabSection(with: uuid),
-            getLibrariesSection(with: uuid),
+            getNewTabSection(with: uuid, tabInfo: tabInfo),
```

Comment:
we should not be doing this, because a) we don't need the whole tab info for one thing, and 2) the resulting actions should not know about stuff they don't need. A better way to do it is that, in `MainMenuMiddleware` you can call the tab manager middleware and get whether it's a homepage or not in there, and then, when that returns, you then call your telemetry call there.